publish-service/src/main/docker/README.txt

```diff
@@ -0,0 +1,16 @@
+A: Build RemRem-Publish Docker image based on RemRem-Publish from an Artifactory, e.g. Jitpack:
+cd (git root dir)/publish-service
+docker build -t remrem-publish:0.6.8 --build-arg URL=https://jitpack.io/com/github/Ericsson/eiffel-remrem-publish/publish-service/0.6.8/publish-service-0.6.8.war -f src/main/docker/Dockerfile .
+
+
+
+B: Build RemRem-Publish based on local RemRem-Publish source code changes
+1. Build RemRem-Publish service artifact:
+cd (git root dir)
+mvn package -DskipTests
+
+2. Build RemRem-Publish Docker image:
```

Comment:
The header here can be more precise; and mention the locally built image (based on latest source code changes).